crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -1817,6 +1817,28 @@ impl Capsule2d {
             half_length: length / 2.0,
         }
     }
+
+    /// Get the part connecting the semicircular ends of the capsule as a [`Rectangle`]
+    #[inline(always)]
+    pub fn to_rectangle(&self) -> Rectangle {
```

Comment:
This naming is somewhat ambiguous. It's unclear at a glance if it's getting the AABB surrounding the full capsule or the inner connective rectangle.

Reply:
I can see the possible confusion although I can't immediately think of a clearer name (maybe `to_inner_rectangle`?). For reference the AABB method is `Capsule2d::aabb_2d`

Reply:
to_inner_rectangle is nice.

Reply:
Renamed to `to_inner_rectangle`